sql/moz-fx-data-shared-prod/forecasts/dataset_metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: Forecasts
+description: |-
+  Views on forecasts based on Firefox data and external models
+dataset_base_acl: view
+user_facing: true
+labels: {}
+default_table_workgroup_access:
```

Comment:
I don't think `default_table_workgroup_access` should be included here. `workgroup_access` should already define all permissions that will be inherited by all queries / datasets / tables defined within this namespace unless explicitly overwritten in their own metadata files.